frontend/src/app/modules/auth/components/auth-card/auth-card.component.ts

```diff
@@ -1,11 +1,41 @@
 import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AuthFormService } from '@core/services/auth-form.service';
+import { NotificationService } from '@core/services/notification.service';
+import { TeamService } from '@core/services/team.service';
+import { TeamMemberService } from '@core/services/team-member.service';
+import { switchMap } from 'rxjs';
 
 @Component({
     selector: 'app-auth-card',
     templateUrl: './auth-card.component.html',
     styleUrls: ['./auth-card.component.sass'],
 })
 export class AuthCardComponent {
+    teamId: number = 0;
+
+    constructor(
+        private route: ActivatedRoute,
+        private teamMemberService: TeamMemberService,
+        private authService: AuthFormService,
+        private teamService: TeamService,
+        private notificationService: NotificationService,
+    ) {
+        this.authService
+            .teamMemberAddedEmitted$
+            .pipe(
+                switchMap(resp => this.teamService.createTeamMember(resp)),
+            )
+            .subscribe({ next: () => {
```

Comment:
I don't unsubscribe. You should inherit this component from BaseComponent and the unse untilThis

Reply:
Fixed it
![image](https://user-images.githubusercontent.com/98499085/191914084-555faaf1-f1a7-42ef-95e5-297845103b5b.png)